myhpom/templates/myhpom/accounts/next_steps.html

```diff
@@ -31,7 +31,7 @@ <h4>
                             <div class="col-md-6 vertical-fill-box">
                                 <div class="advance-directive-block">
                                     <h5>
-                                        <a href="{{ ad_template.advance_directive_template.url }}">
+                                        <a href="{{ ad_template.url }}" target="_blank">
```

Comment:
Welp, I'm having the same problem you're having, and it looks to me like the media configuration isn't right:

 * I think we need to set up media files during development: https://docs.djangoproject.com/en/1.8/howto/static-files/#serving-files-uploaded-by-a-user-during-development

 * And we also need to change the `MEDIA_ROOT` setting, which is inside the STATIC_URL (which is a no no -- see the warning here: https://docs.djangoproject.com/en/1.8/ref/settings/#media-root)

 * for production/staging we'll want to tweak the nginx configurations(nginx/config-files/hydroshare-nginx.conf#21 and nginx/config-files/hydroshare-ssl-nginx.conf#30).

Reply:
One other little thing here - this needs an `rel="noopener noreferrer"`